src/components/entryTab.tsx

```diff
@@ -110,6 +111,14 @@ export function EntryTab(props: {
     for (const entity of props.entities) {
         const course = entity.getCourse();
         for (const entry of entity.entries) {
+            const hasfinished = entry.hasFinished;
+            if (hasfinished == true){
+                checkedElements.push({
+                    entry: entry,
+                    course: course
+                });
+                break;
```

Comment:
[must]
ここの`break;`によってコースサイトの課題で1つでも完了済みのものがあるとそれ以降の課題が無視されてしまいます．
なので特段意図がなければ，
```suggestion
                continue;
```
に変更して欲しいです 🙇

Reply:
わかりました。ありがとうございます！